### PR TITLE
Fix undefined meta transaction type for SCW

### PR DESCRIPTION
**Pull request by NotRealAI**
Quick fix for #67 
Biconomy sdk inits fine after this check

It seems that wherever `smartContractMetaTransactionMap` is used for SCW it has an address from `api`, so moving it inside the branch seems fine.

Built files NOT included here because my particular node version was causing a huge diff in dist/mexa.js, someone on the team will have to build probably.

### CODE REVIEW

**Review by livingrock7 — COMMENTED** [review]
Looks good. Will test locally and update along with build files.

**Comment by NotRealAI** [comment]
Changed to integration branch.

By the way, I'm not certain where the difference in build comes from, but I suspect it's node version. If you add an `engines` field to your package.json it indicates to developers which version to use. It's informational metadata only, it doesn't actually change anything about the build.
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines

**Comment by NotRealAI** [comment]
I was incorrect in my assumption, `api.contractAddr` doesn't include anything for the SCW. Had to make changes to use the `config.SCW` for smart contract wallet.

**Review by livingrock7 — COMMENTED** [review]
can you confirm based on below review commen please L307

File: src/Biconomy.js [hunk 1]
```diff
@@ -299,10 +299,15 @@ Biconomy.prototype.getForwardRequestAndMessageToSign = function (
         let api = engine.dappAPIMap[to]
           ? engine.dappAPIMap[to][methodName]
           : undefined;
+        let metaTxApproach
         if (!api) {
           api = engine.dappAPIMap[config.SCW]
             ? engine.dappAPIMap[config.SCW][methodName]
             : undefined;
+          metaTxApproach[config.SCW]
```
[the remark this answers is not in the document]
Comment (NotRealAI) [reply src/Biconomy.js:307]: Oh thank you for catching that! Absolutely just a typo, my code usage wasn't hitting this branch so I missed it. Will resolve.

**Pull request merged**